src/formatterLoader.ts

```diff
@@ -52,24 +52,24 @@ export function findFormatter(name: string | FormatterFunction, formattersDirect
     }
 }
 
-function loadFormatter(...paths: string[]) {
+function loadFormatter(...paths: string[]): FormatterConstructor | undefined {
     const formatterPath = paths.reduce((p, c) => path.join(p, c), "");
     const fullPath = path.resolve(moduleDirectory, formatterPath);
 
     if (fs.existsSync(`${fullPath}.js`)) {
         const formatterModule = require(fullPath);
-        return formatterModule.Formatter;
+        return formatterModule.Formatter; // tslint:disable-line no-unsafe-any
```

Comment:
instead of disabling rule, cast the `require` to an interface like  `{ Formatter: FormatterConstructor }`